<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EX2W</class>
 <widget class="QWidget" name="EX2W">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>624</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1089</width>
    <height>624</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1089</width>
    <height>624</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EX2W</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>1091</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="chartLayout"/>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>1091</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Maksymalna liczba węzłów: 2^</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Dolna granica całkowania:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Wykres:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QPushButton" name="computeAndWriteButton">
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Rysuj wykres funkcji  i oblicz wartości</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Górna granica całkowania:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QTextEdit" name="nodeNmbrTextEdit">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QTextEdit" name="toTextEdit">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="chartsComboBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QTextEdit" name="fromTextEdit">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>1091</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="4">
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Wynik metody Simpsona</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="analiticTextBrowser">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QTextBrowser" name="simpsonTextBrowser">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Wynik analityczny : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Wynik metody trapezów: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QTextBrowser" name="trapezTextBrowser">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="chartsComboBox_2">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Wykres funkcji</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wykres dokładnośći metodą trapezów</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wykres dokładności metodą Simpsona</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>519</y>
     <width>1091</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Dokładność wykresu:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="chartPrecisionTextEdit">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1080&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_9">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Wzór funkcji:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
